spec/lib/embed/viewer/common_viewer_spec.rb

```diff
@@ -72,10 +72,10 @@
   end
   describe 'footer_html' do
     it 'should return the objects footer' do
-      stub_request(request)
+      stub_purl_response_and_request(file_purl, request)
       html = Capybara.string(file_viewer.footer_html)
       expect(html).to have_css 'div.sul-embed-footer'
-      expect(html).to have_css '[aria-label]', count: 2
+      expect(html).to have_css '[aria-label]', count: 3
```

Comment:
a better test would look for the values in the aria-label;  this is pretty generic.

Also, it isn't testing for the VALUE displaying, only the label.